react-frontend/src/components/CoCos/coCoCard.js

```diff
@@ -54,7 +54,7 @@ export const Badges = (status, maintenanceStatus, tags, colour) => {
       )}
       {/* Only display 3 tags */}
       {tags?.map((tag, i) => {
-        if (i > 2) {
+        if (i > 10) {
```

Comment:
Not sure if this worked? I was able to populate 11 tags on a card:

![Screen Shot 2021-09-23 at 9 06 27 AM](https://user-images.githubusercontent.com/16946297/134543982-d87b7888-76fc-4cbb-9866-3a82044445c4.png)

However, the likelihood of this becoming a problem (of the content manager trying to attach way too many tags to a CoCo) is very low.

Reply:
Oh that's because counting starts at 0 in javascript.  Good catch.